include/swift/Driver/Compilation.h

```diff
@@ -160,6 +160,11 @@ class Compilation {
   /// of date.
   bool EnableIncrementalBuild;
 
+  /// When true, emit duplicated compilation record file with filename suffixed
+  /// with '~moduleonly'.  This is used for '-emit-module' only incrematal
+  /// builds.
```

Comment:
Would it be possible to explain a bit more in this comment? *Why* is it necessary to add that suffix?
Also (grammar nitpick): "emit a duplicated compilation record file whose filename is suffixed with". 
And the second sentence could be clearer: this what? This suffix? This mode?
Thanks!